app/controllers/catalog_controller.rb

```diff
@@ -13,6 +13,14 @@ class CatalogController < ApplicationController
     BotChallengePage::BotChallengePageController.bot_challenge_enforce_filter(controller, immediate: true)
   end
 
+  before_action only: :index do
```

Comment:
Can you put a comment on this saying that Bento calls the JSON API and needs the icon info?